tests/test_exceptions.py

```diff
@@ -9,8 +9,52 @@
 import unittest
 import inspect
 
+import mock
+
 import tests
 import ga4gh.frontend_exceptions as frontendExceptions
+import ga4gh.frontend as frontend
+
+
+class TestExceptionHandler(unittest.TestCase):
+    """
+    Test that caught exceptions are handled correctly
+    """
+    class UnknownException(Exception):
+        pass
+
+    class DummyFlask(object):
+
+        def __call__(self, *args):
+            return TestExceptionHandler.DummyResponse()
+
+    class DummyResponse(object):
+        pass
+
+    def run(self, *args, **kwargs):
+        # patching is required because flask.jsonify throws an exception
+        # if not being called in a running app context
+        dummyFlask = self.DummyFlask()
+        with mock.patch('flask.jsonify', dummyFlask):
+            super(TestExceptionHandler, self).run(*args, **kwargs)
+
+    def testMappedException(self):
+        originalExceptionClass, mappedExceptionClass = \
```

Comment:
Why not loop over all of them? map doesn't return a list in py3, so this will break then.